.github/workflows/upload.yml

```diff
@@ -35,13 +35,24 @@ jobs:
         env:
           CI: false
       - name: Compress
-        run: cd build && tar -czf ../${{env.PACKAGE_NAME}} ./
-      - name: Install upload-assets snap
-        run: sudo snap install upload-assets
+        run: cd build && tar -czf ../${{env.PACKAGE_NAME}} ./ && ls -hs ../${{env.PACKAGE_NAME}}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install deps
+        run: pip install requests
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
this is weird, and duplicates installing Python for reasons that are not immediately apparent

Reply:
Good spot, but this will be removed as soon as upload-script is fixed.